lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -28,7 +28,7 @@ boltons==19.3.0
 boto3==1.9.114
 boto==2.49.0
 botocore==1.12.253
-bx-python==0.8.6
+bx-python==0.8.7
```

Comment:
@mvdbeek Should I revert this for the time being given https://github.com/bxlab/bx-python/pull/58 ?

Reply:
Yeah, I'll take care of this in the converter PR (which also tests bx-python).